src/__tests__/BrahmosPureComponent.test.js

```diff
@@ -0,0 +1,70 @@
+/*
+Inspired by -
+https://github.com/facebook/react/blob/master/packages/react/src/__tests__/ReactPureComponent-test.js
+*/
+import Brahmos, { render } from "..";
+
+describe("BrahmosPureComponent", () => {
+  it("should render", () => {
+    let renders = 0;
+    class Component extends Brahmos.PureComponent {
```

Comment:
You can keep the component creation outside of the test case.

Reply:
@s-yadav This component is used only in this test case so should I move the creation of only this component or both the components in both the test cases outside the test cases.

Reply:
The Component creation can be reused across the specs of PureComponent spec. So it's better to keep it outside.